src/components/view/Garage/Track/track.ts

```diff
@@ -0,0 +1,162 @@
+import './track.css';
+import ICar from '../../../controller/ICar';
+
+class Track {
+    amountOfCars: number;
+    selectedID: number;
+    amountOfPages: number;
+    currentPage: number;
+    garage: ICar[];
+    constructor() {
+        const app = <HTMLDivElement>document.getElementById('app');
+        const garageWrapper = document.createElement('div');
+        garageWrapper.className = 'garage_wrapper';
+
+        const h2 = document.createElement('h2');
+        h2.textContent = 'Garage (0)';
+        const h3 = document.createElement('h3');
+        h3.textContent = 'Page #1';
+
+        const trackWrapper = document.createElement('div');
+        trackWrapper.className = 'track_wrapper';
+
+        const pagbuttonsWrapper = document.createElement('div');
+        pagbuttonsWrapper.className = 'pagination_buttons_wrapper';
+        const prevButton = document.createElement('button');
+        prevButton.className = 'prev_btn';
+        prevButton.textContent = 'PREV';
+        const nextButton = document.createElement('button');
+        nextButton.className = 'next_btn';
+        nextButton.textContent = 'NEXT';
+        pagbuttonsWrapper.append(prevButton);
+        pagbuttonsWrapper.append(nextButton);
+
+        garageWrapper.append(h2);
+        garageWrapper.append(h3);
+        garageWrapper.append(trackWrapper);
+        garageWrapper.append(pagbuttonsWrapper);
+        app.append(garageWrapper);
+        this.amountOfCars = 0;
+        this.amountOfPages = 0;
+        this.garage = [];
+        this.currentPage = 0;
+        this.selectedID = -1;
+    }
+
+    drawCars(data: ICar[], addListeners: (carElement: HTMLDivElement) => void) {
```

Comment:
this method not only draws a car but sets data passed through arguments, generates car, adds listeners,